src/server/endpoints/sensors.py

```diff
@@ -1,95 +1,43 @@
-from flask import Blueprint, Request, request
+from flask import Blueprint, request
 
-from .._config import AUTH, GRANULAR_SENSOR_AUTH_TOKENS, OPEN_SENSORS
 from .._exceptions import EpiDataException
 from .._params import (
     extract_strings,
     extract_integers,
 )
+from .._security import current_user
+from .._config import GRANULAR_SENSOR_ROLES, OPEN_SENSORS
 from .._query import filter_strings, execute_query, filter_integers
-from .._validate import (
-    require_all,
-    resolve_auth_token,
-)
+from .._validate import require_all
 from typing import List
 
 # first argument is the endpoint name
 bp = Blueprint("sensors", __name__)
 alias = "signals"
 
-#   Limits on the number of effective auth token equality checks performed per sensor query; generate auth tokens with appropriate levels of entropy according to the limits below:
-MAX_GLOBAL_AUTH_CHECKS_PER_SENSOR_QUERY = 1  # (but imagine is larger to futureproof)
-MAX_GRANULAR_AUTH_CHECKS_PER_SENSOR_QUERY = 30  # (but imagine is larger to futureproof)
-#   A (currently redundant) limit on the number of auth tokens that can be provided:
-MAX_AUTH_KEYS_PROVIDED_PER_SENSOR_QUERY = 1
-# end sensor query authentication configuration
-
-PHP_INT_MAX = 2147483647
-
-
-def _authenticate(req: Request, names: List[str]):
-    auth_tokens_presented = (resolve_auth_token(req) or "").split(",")
 
+def _authenticate(names: List[str]):
```

Comment:
todo: the old `_authenticate` had two arguments; looks like there are still some calls that include the request (line 57) that should be fixed

Reply:
while you're at it you can probably drop the names argument as well, since the first line of the function overwrites it

Reply:
already fixed!